common/common_target.mk

```diff
@@ -50,6 +50,8 @@ else
 	UNAME:=$(shell uname -s)
 	ifeq ($(UNAME), Linux)
 		OS:=linux
+	else
```

Comment:
I had originally considered that, but I think this section needs to be reworked over all (essentially making the Windows case fall in line with the pattern as well).